docs/quick-start.rst

```diff
@@ -1,41 +0,0 @@
-:hide-navigation:
```

Comment:
Please add a redirect for the page we are removing.

Reply:
Already present [here](https://github.com/robusta-dev/robusta/pull/1246/files#diff-85933aa74a2d66c3e4dcdf7a9ad8397f5a7971080d34ef1108296a7c6b69e7e3R82)